packages/web-service/src/pages/habitat/activities/habitat-activities.js

```diff
@@ -1,18 +1,58 @@
+import Joi from 'joi'
 import pageRoute from '../../../routes/page-route.js'
 import { habitatURIs } from '../../../uris.js'
+import { settDistruptionMethods } from '../../../utils/sett-disturb-methods.js'
 import { PowerPlatformKeys } from '@defra/wls-powerapps-keys'
-const { METHOD_IDS: { DAMAGE_A_SETT, DESTROY_A_SETT, DISTURB_A_SETT, OBSTRUCT_SETT_WITH_BLOCK_OR_PROOF, OBSTRUCT_SETT_WITH_GATES } } = PowerPlatformKeys
+import { APIRequests } from '../../../services/api-requests.js'
+import { changeHandler, putData } from '../../../utils/editTools.js'
+const page = 'habitat-activities'
+
+const {
+  SPECIES: { BADGER },
+  ACTIVITY_ID: { INTERFERE_WITH_BADGER_SETT }
+} = PowerPlatformKeys
 
 export const completion = async _request => habitatURIs.CHECK_YOUR_ANSWERS.uri
 
 export const getData = async _request => {
-  return {
-    OBSTRUCT_SETT_WITH_GATES,
-    OBSTRUCT_SETT_WITH_BLOCK_OR_PROOF,
-    DAMAGE_A_SETT,
-    DESTROY_A_SETT,
-    DISTURB_A_SETT
+  return { settDistruptionMethods }
```

Comment:
Again - please all the page text in the *.njk files

Reply:
Resolved